pictures/compute_pi/.gitattributes

```diff
@@ -0,0 +1,5 @@
+*.mp4 filter=lfs diff=lfs merge=lfs -text
```

Comment:
Do we really need this file or is it enough to add mp4 and jpeg to the central .gitattributes file?